src/components/Header/index.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import { Link as GatsbyLink } from "gatsby";
+import Button from "@rescui/button";
+import { MobileMenu } from "./mobile-menu/";
```

Comment:
Oh... is it possible with ends`/`? O_o
wow

Reply:
`/` removed